components/eamxx/src/physics/rrtmgp/eamxx_rrtmgp_process_interface.cpp

```diff
@@ -697,13 +874,112 @@ void RRTMGPRadiation::run_impl (const double dt) {
       auto cld_tau_lw_bnd  = subview_3d(m_buffer.cld_tau_lw_bnd);
       auto cld_tau_sw_gpt  = subview_3d(m_buffer.cld_tau_sw_gpt);
       auto cld_tau_lw_gpt  = subview_3d(m_buffer.cld_tau_lw_gpt);
+#endif
+#ifdef RRTMGP_ENABLE_KOKKOS
+      // If YAKL is on, we don't want aliased memory in both the yakl and kokos
+      // subviews.
+      auto subview_1dk = [&](const real1dk v) -> real1dk {
+        real1dk subv(v, std::make_pair(0, ncol));
+#ifdef RRTMGP_ENABLE_YAKL
+        real1dk rv(v.label(), ncol);
```

Comment:
I don't think we need to deep copy here. I was really just taking a subview, so that all the data copy down below could be done.

Reply:
We do need a deep copy if YAKL is enabled:
```
      // If YAKL is on, we don't want aliased memory in both the yakl and kokos                                                                                                                                                 
      // subviews.
```

Reply:
Why do you need them deep copied though? What's wrong with an unmanaged one?

Reply:
Oh, you're assuming there's never anything in the input view that would be used later? It's possible that's true but I didn't verify and I don't think it hurts anything.

We can't use unmanaged views because RRTMGP is not currently coded to accept those.